include/logging/log_backend.h

```diff
@@ -94,7 +95,25 @@ static inline void log_backend_put(const struct log_backend *const backend,
 }
 
 /**
- * @brief Function for reconfiguring backend to panic mode.
+ * @brief Notify backend about dropped log messages.
+ *
+ * Function is optional.
+ *
+ * @param[in] backend  Pointer to the backend instance.
+ * @param[in] cnt      Number of dropped logs since last notification.
+ */
+static inline void log_backend_dropped(const struct log_backend *const backend,
+				       u32_t cnt)
+{
+	assert(backend);
```

Comment:
Please use `__ASSERT_NO_MSG`

Reply:
done, created dedicated commit for cleanup (doxygen + asserts)